e2e/tests/odoo-openmrs-flows.spec.ts

```diff
@@ -139,7 +139,7 @@ test('Revising a synced drug order edits corresponding quotation line in Odoo',
   await expect(drugOrderItem).toContainText('Thrice daily - 6 Days');
 });
 
-test('Discontinuing a synced drug order cancels corresponding quotation line in Odoo', async ({ page }) => {
+test('Discontinuing a synced OpenMRS drug order cancels the corresponding Odoo quotation.', async ({ page }) => {
```

Comment:
Surely this is not what's being tested right? What if there're other items on that quotation?

Reply:
> Discontinuing a synced OpenMRS drug order cancels the corresponding Odoo quotation.

The test considers a scenario when there is only one item on the quotation.

> What if there're other items on that quotation?

Only the discontinued synced drug order gets deleted in the quotations table and the other Odoo quotation line remain active.

Reply:
Shouldn't it be something like 
>Discontinuing a synced OpenMRS drug order removes the corresponding Odoo quotation line.

?

Reply:
It makes sense, though I need to add some additional tests to assert that the Odoo quotation line gets removed.

Reply:
Looks like there's more to unpack here and this may be worth a new ticket:

- Voiding the drug order should make the quotation line go away, that's the bottom line.
- If this was the only thing on the quotation... well this should probably cancel the quotation indeed.

Reply:
We can add a new test case for discontinuing a synced drug order for a customer with multiple quotation lines. This can be considered in a new ticket.

Reply:
Logged a ticket at https://mekomsolutions.atlassian.net/browse/OZ-553